go.mod

```diff
@@ -18,9 +18,10 @@ require (
 	github.com/containerd/typeurl v0.0.0-20190228175220-2a93cfde8c20
 	github.com/containernetworking/plugins v0.7.6
 	github.com/davecgh/go-spew v1.1.1
-	github.com/docker/distribution v2.7.1+incompatible
+	github.com/docker/distribution v2.7.2-0.20200311034821-a45a401e9709+incompatible
```

Comment:
This one should no longer be needed since https://github.com/containerd/containerd/pull/4251 and https://github.com/containerd/cri/pull/1484

Do you know what's using it?